.github/workflows/windows.yml

```diff
@@ -561,12 +561,12 @@ jobs:
           tflags: '~1516 ~2301 ~2302 ~2303 ~2307'
           config: '-DENABLE_DEBUG=ON -DENABLE_UNICODE=OFF -DCURL_USE_SCHANNEL=OFF -DCURL_BROTLI=ON -DCURL_ZSTD=ON -DCURL_USE_LIBPSL=ON -DUSE_NGHTTP2=ON -DBUILD_SHARED_LIBS=OFF -DCURL_USE_LIBSSH2=ON -DCURL_USE_OPENSSL=ON -DUSE_HTTPSRR=ON -DUSE_ECH=ON'
         - name: '!ssl'
-          install: 'brotli zlib zstd libpsl'
+          install: 'brotli zlib zstd libpsl pkgconf libidn2'
           arch: 'x64'
           plat: 'windows'
           type: 'Debug'
-          tflags: '~1516 ~2301 ~2302 ~2303 ~2307'
-          config: '-DENABLE_DEBUG=ON -DENABLE_UNICODE=OFF -DCURL_USE_SCHANNEL=OFF -DCURL_BROTLI=ON -DCURL_ZSTD=ON -DCURL_USE_LIBPSL=ON -DHTTP_ONLY=ON'
+          tflags: '~1516 ~2301 ~2302 ~2303 ~2307 ~165 ~1448 ~2046 ~2047'
```

Comment:
What do you think about doing this in the test run step?:
```shell
         if [[ '${{ matrix.config }}' = *'-DUSE_WIN32_IDN=ON'* || \
                '${{ matrix.config }}' = *'-DUSE_LIBIDN2_IDN=ON'* ]]; then
```

Reply:
ok :)

Reply:
But the tests ~1448 ~2046 ~2047 are failed with -DENABLE_UNICODE=OFF as well. Then I will need to remove the second if, and add them anyway.

Reply:
See my other comment.

libidn2 seems to break IDN differently than WinIDN, in particular 165 fails with this:
```
There was no content at all in the file log/8/server.input.
```